src/gsi/gfs_stratosphere.f90

```diff
@@ -1153,11 +1155,17 @@ subroutine add_gfs_stratosphere
          ncdim = get_dim(atmges, 'grid_xt'); lonb = ncdim%len
          ncdim = get_dim(atmges, 'grid_yt'); latb = ncdim%len
          ncdim = get_dim(atmges, 'pfull'); levs = ncdim%len
+         write(6,*) 'ncio lonb latb lev =',lonb, latb, levs
          njcap = -9999
+         ! FV3GFS write component does not include JCAP, infer from DIMY-2
+         if (njcap<0) njcap=latb-2
          jcap_org = njcap
+         write(6,*) 'ncio njcap =',jcap_org, njcap
          idate6 = get_idate_from_time_units(atmges)
+         write(6,*) 'ncio idate =',idate6
```

Comment:
These prints, either remove, or cleanup and add inside if (mype==0)

Reply:
Cleaned up.